static/js/components/PrivacyTab.js

```diff
@@ -25,6 +26,12 @@ class PrivacyTab extends ProfileTab {
     }
   };
 
+  saveAndContinue = () => {
+    saveAndContinue.call(this, null, true).then(() => {
+      this.context.router.push(this.nextUrl);
+    });
```

Comment:
I added an instance method ('saveAndContinue`) to `ProfileTab` that we might want to use instead here - we could refactor that method to take an optional boolean to signify it's the last tab (something like `lastTab`? haha probs not a good name).